targets/somm

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e -v -x
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+REQUIRES='core'
+PROVIDES='x11'
+DESCRIPTION='Wayland backend running in a Chromium OS window.'
+HOSTBIN='startsommelier'
+CHROOTBIN='runsommelier'
+CHROOTETC='sommelierrc'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+XMETHOD="${XMETHOD:-sommelier}"
+
+# Download the latest sommelier
+urlbase="https://chromium.googlesource.com/chromiumos/platform2/+archive/fef7f012df0e5ec865f94255c10503ec24d9a370/vm_tools"
+
+# Download the corresponding virtwl.h
+urlvirtwl="https://chromium.googlesource.com/chromiumos/third_party/kernel/+/7f7629cc680bf3c22d547fb5333e521af58c6c9e/include/uapi/linux/virtwl.h?format=TEXT" 
+
+DUMMYBUILDTMP="`mktemp -d crouton-cras.XXXXXX --tmpdir=/tmp`"
+
+addtrap "rm -rf --one-file-system '$DUMMYBUILDTMP'"
+
+echo "Download sommelier..." 1>&2
+
+superinsecure=''
+if release -le wheezy; then
+     # Wheezy doesn't have the right CA in ca-certificates that validates x.org
```

Comment:
is the `superinsecure` hack necessary for chromium?

Reply:
commit e9f9710994196c06f21f29bb708ec194df618433